api/handler/resolution.go

```diff
@@ -901,3 +902,80 @@ func UpdateResolutionStepState(c *gin.Context, in *updateResolutionStepStateIn)
 
 	return nil
 }
+
+type resolveTemplatingResolutionIn struct {
+	PublicID    string `path:"id" validate:"required"`
+	TemplateStr string `json:"template_str" validate:"required"`
```

Comment:
I suggest renaming `TemplateStr` to `Template`, as the type is already exposed.

Reply:
I think using `Template` might be confusing with the `tasktemplate`/`template_name` used a lot in µTask.
I suggest `TemplatingExpression`, what do you think ?

Reply:
Sounds good for `TemplatingExpression`, with field named `template_expr` maybe ?